lib/dalli/pid_cache.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Dalli
+  module PIDCache
```

Comment:
@casperisfine  Just add the below above the `module PIDCache` line and the problem should be resolved.  Once that's done I'll merge.

```
##
# Dalli::PIDCache is a wrapper class for PID checking to avoid system calls when checking the PID.
##
```

Reply:
Done